doc/stdenv/stdenv.chapter.md

```diff
@@ -1375,6 +1375,26 @@ This hook only runs when compiling for Linux.
 
 This sets `SOURCE_DATE_EPOCH` to the modification time of the most recent file.
 
+### `add-bin-to-path.sh` {#add-bin-to-path.sh}
+
+This setup hook adds the `./bin/` directory to the `PATH` environment variable,
```

Comment:
This should probably at least explicitly mention it's adding `$out/bin` and not `$sourceRoot/bin` or whatever. Honestly, this is my big concern with this hook: it's not immediately obvious what exactly it does, and the thing it does comes with a bunch of weird caveats (what if there are multiple outputs? what if the binaries are in sbin? etc).